tests/func/test_get.py

```diff
@@ -125,9 +126,24 @@ def test_get_full_dvc_path(erepo):
     assert filecmp.cmp(external_data, "ext_data_imported", shallow=False)
 
 
+def test_non_cached_output(request, erepo):
+    os.chdir(erepo.root_dir)
```

Comment:
It's better to use pytests `monkeypatch.chdir()`, it will automatically undo that at the end of the test.

Reply:
`monkeypatch.chdir()` attempts to reverse the directory changes after the test. This fails for directories that we entered and deleted during the test

```
monkeypatch.chdir(tmp_path)
Repo.get(erepo.root_dir, "non_cached_file")
```

```
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/dvctest.29427.8d8awfm3.92iSUjV4TyDafrpFVsWjiX'
```

Reply:
Oh, I see, let's leave it as is for now. This might be cleaned up once https://github.com/iterative/dvc/pull/2868 lands.